Makefile

```diff
@@ -121,51 +131,32 @@ feedernode: juno-cached
 	--p2p \
 	--p2p-feeder-node \
 	--p2p-addr=/ip4/0.0.0.0/tcp/7777 \
-	--p2p-private-key="5f6cdc3aebcc74af494df054876100368ef6126e3a33fa65b90c765b381ffc37a0a63bbeeefab0740f24a6a38dabb513b9233254ad0020c721c23e69bc820089" \
+	--p2p-private-key=$(FEEDER_KEY) \
+	--disable-l1-verification
 
-node1: juno-cached
+define run_node
 	./build/juno \
 	--network=sepolia \
 	--log-level=debug \
-	--metrics \
-	--db-path=./p2p-dbs/node1 \
+	--db-path=./p2p-dbs/$(1) \
 	--p2p \
-	--p2p-peers=/ip4/127.0.0.1/tcp/7777/p2p/12D3KooWLdURCjbp1D7hkXWk6ZVfcMDPtsNnPHuxoTcWXFtvrxGG \
-	--p2p-addr=/ip4/0.0.0.0/tcp/7778 \
-	--p2p-private-key="8aeffc26c3c371565dbe634c5248ae26f4fa5c33bc8f7328ac95e73fb94eaf263550f02449521f7cf64af17d248c5f170be46c06986a29803124c0819cb8fac3" \
-	--metrics-port=9091
+	--p2p-peers=$(2) \
+	--p2p-private-key=$(3) \
+	--metrics-port=$(4) \
+	--disable-l1-verification
+endef
 
-#	--p2p-peers=/ip4/127.0.0.1/tcp/7778/p2p/12D3KooWDQVMmK6cQrfFcWUoFF8Ch5vYegfwiP5Do2SFC2NAXeBk \
+node1: juno-cached ## Run a node
+	$(call run_node,node1,$(FEEDER_ADDR),$(NODE1_KEY),9091)
 
-node2:
-	./build/juno \
-	--network=sepolia \
-	--log-level=debug \
-	--db-path=./p2p-dbs/node2 \
-	--p2p \
-	--p2p-peers=/ip4/127.0.0.1/tcp/7777/p2p/12D3KooWLdURCjbp1D7hkXWk6ZVfcMDPtsNnPHuxoTcWXFtvrxGG \
-	--p2p-private-key="2d87e1d1c9d8dda1cf9a662de1978d2cd0b96e6ba390c75ded87c6c4fab657057fa782ae5977c3bd02d58281dccd16f2c26990d1f6c22f818a84edac97957348" \
-	--metrics-port=9092
+node2: juno-cached ## Run a node
+	$(call run_node,node2,$(FEEDER_ADDR),$(NODE2_KEY),9092)
 
-node3:
-	./build/juno \
-	--network=sepolia \
-	--log-level=debug \
-	--db-path=./p2p-dbs/node3 \
-	--p2p \
-	--p2p-peers=/ip4/127.0.0.1/tcp/7777/p2p/12D3KooWLdURCjbp1D7hkXWk6ZVfcMDPtsNnPHuxoTcWXFtvrxGG \
-	--p2p-private-key="54a695e2a5d5717d5ba8730efcafe6f17251a1955733cffc55a4085fbf7f5d2c1b4009314092069ef7ca9b364ce3eb3072531c64dfb2799c6bad76720a5bdff0" \
-	--metrics-port=9093
+node3: juno-cached ## Run a node
+	$(call run_node,node3,$(FEEDER_ADDR),$(NODE3_KEY),9093)
 
-pathfinder: juno-cached
-	./build/juno \
-    	--network=sepolia \
-    	--log-level=debug \
-    	--db-path=./p2p-dbs/node-pathfinder \
-    	--p2p \
-    	--p2p-peers=/ip4/127.0.0.1/tcp/8888/p2p/12D3KooWF1JrZWQoBiBSjsFSuLbDiDvqcmJQRLaFQLmpVkHA9duk \
-    	--p2p-private-key="54a695e2a5d5717d5ba8730efcafe6f17251a1955733cffc55a4085fbf7f5d2c1b4009314092069ef7ca9b364ce3eb3072531c64dfb2799c6bad76720a5bdff0" \
-    	--metrics-port=9094
+pathfinder: juno-cached ## Run a node to sync from pathfinder
+	$(call run_node,pathfinder,$(PATHFINDER_ADDR),$(PATHFINDER_KEY),9094)
```

Comment:
In my opinion we should keep it as it was. These commands are temporary for our convinience to test p2p

Reply:
> we should keep it as it was

node1, node2, node3, pathfinder targets